test/templates/tile/index.njk

```diff
@@ -153,14 +127,14 @@
 
     <br><br>
     <h2>disabled</h2>
-    <p>Add <code>aria-disabled=true</code></p>
+    <p>Add <code>is-disabled</code></p>
     {% set example %}
-{%- call tile(aria={disabled: true}) -%}
+{%- call tile() -%}
```

Comment:
These examples no longer appear disabled.  I think a parameter will need to be passed, but just not an aria attribute.